apps/gauzy/src/app/pages/accounting-templates/accounting-templates.component.scss

```
@@ -1,25 +1,67 @@
+// exports no Sass members) and must stay.
 @use 'themes' as *;
 @use '../email-templates/email-templates.component' as *;
 
+// This page carried two `!important` height overrides — `calc($height + 3.5rem)`
+// on the Ace editor and `calc($height + 1.3rem)` on the preview box — which
+// added even more to a `100vh` constant that already ignored the footer, which
+// is why the overlap showed up worst here. Both are gone: email-templates.scss
+// now sizes the two panels from the content column via flex, and that shared
+// chain also covers this page's one-editor layout.
 :host {
+  // A settings page CANNOT set its own `display`. `pages.component.scss` carries a
```

Stylelint (17.14.1)

[error] Unknown word A (CssSyntaxError)

(CssSyntaxError)